src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_product.yml

```diff
@@ -20,3 +20,6 @@ sylius_product:
                 repository: Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductVariantRepository
                 form:
                     default: Sylius\Bundle\CoreBundle\Form\Type\Product\ProductVariantType
+        product_option:
+            classes:
+                repository: Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductOptionRepository
```

Comment:
Should, but it won't replace the class name inside of Doctrine itself, only for the service if we configure it via xml file in the bundle. This is that old annoying bug, otherwise we cannot use custom methods in grids because they use ``getRepository``.